color/Tiltfile

```diff
@@ -0,0 +1,74 @@
+RESET='\033[0m'
+
+BLACK='\033[0;30m'
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[0;33m'
+BLUE='\033[0;34m'
+magenta='\033[0;35m'
```

Comment:
How come this one isn't capitalized?

Reply:
Because I don't know how to use find and replace 🤦 

(Good catch 😅)